crates/astria-sequencer/src/proposal/commitment.rs

```diff
@@ -121,13 +121,17 @@ mod test {
         };
 
         let signing_key = SigningKey::new(OsRng);
-        let tx = UnsignedTransaction {
-            params: TransactionParams::builder()
-                .nonce(0)
-                .chain_id("test-chain-1")
-                .build(),
-            actions: vec![sequence_action.clone().into(), transfer_action.into()],
-        };
+
+        let tx = UnsignedTransaction::builder()
```

Comment:
can you make a utility for this? It looks like there is a lot of repetition for constructing these transactions here.